modules/ROOT/pages/apicat-about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,42 @@
+= Autocataloging APIs Using API Catalog CLI
+
+API Catalog CLI enables you to discover and catalog your API specifications, documentation files, and associated metadata as part of an automated process. You can embed the publish asset command in your automation tools, such as a CI/CD pipeline or custom scripts, to automatically trigger the cataloging of your API assets. API Catalog CLI is agnostic of CI/CD tools and runtime environments.
+
+With API Catalog CLI you can:
+
+* Identify all APIs in a project directory structure and create a <<apicat-descriptor-file,descriptor file>> to use to publish the APIs to Exchange.
+* Identify all new and changed APIs and update the descriptor file on an ongoing basis. 
+* <<apicat-trigger-cond,Conditionally trigger>> the publish based on criteria such as branches and tags.
+* <<apicat-version-strategy-cond,Conditionally set the asset version strategy>> based on criteria such as branches and tags.
+* Leverage API metadata in `exchange.json` files as part of the descriptor information if you already have those for your projects. 
+* Publish APIs described in the descriptor file by running commands at a command prompt or in a CI/CD pipeline or custom scripts.
+
+== API Catalog CLI Concepts
+
+Following are the concepts you need to know to use API Catalog CLI.
```

Comment:
```suggestion
Review the following concepts before using API Catalog CLI.
```

Reply:
I wasn't sure if "Review" was the right thing, since it's the first time they will have seen this text. I guess we don't consider it "re-view"?